core/src/test/java/dk/alexandra/fresco/suite/dummy/arithmetic/DummyArithmeticRunner.java

```diff
@@ -0,0 +1,61 @@
+package dk.alexandra.fresco.suite.dummy.arithmetic;
+
+import dk.alexandra.fresco.framework.Application;
+import dk.alexandra.fresco.framework.TestFrameworkException;
+import dk.alexandra.fresco.framework.TestThreadRunner;
+import dk.alexandra.fresco.framework.builder.numeric.ProtocolBuilderNumeric;
+import dk.alexandra.fresco.suite.dummy.arithmetic.AbstractDummyArithmeticTest;
+import dk.alexandra.fresco.suite.dummy.arithmetic.DummyArithmeticResourcePool;
+
+import static dk.alexandra.fresco.framework.sce.evaluator.EvaluationStrategy.SEQUENTIAL;
+
+public class DummyArithmeticRunner {
+    static public <T> T run(Application<T, ProtocolBuilderNumeric> application) {
+        return run(application, 1);
+    }
+
+    static public <T> T run(Application<T, ProtocolBuilderNumeric> application, int numberOfParties) {
```

Comment:
Suggestion. Change run to:
 run(
  Supplier<InputT> input,
  Function<InputT, Application<OutputT> test,
  Consumer<OutputT> verifier
)

This will make the tests stepwise present lambdas with different purposes